researcher_ids/RI_Claims_V1.md

```diff
@@ -60,200 +76,281 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+    process such information.
+
+6.  <a name="requirement-6"></a> All Passport Visas within the
+    `ga4gh_passport_v1` scope eligible for release to the requestor MUST be
+    consistently provided. Reasons for limiting exchange may include user
+    approval, contractual limitations, regulatory restrictions, or filtering
+    claims to only the subset needed for a particular purpose, etc.
+
+7.  <a name="requirement-7"></a> The [Passport JWT Claim](#passport-jwt-claim)
+    is only included in the Passport if the `ga4gh_passport_v1` scope is
+    requested from the [Passport Broker](#passport-broker) and other conditions
+    are met as outlined within this specification.
+
+    If the Passport Broker is the [Passport Visa
+    Signatory](#passport-visa-signatory), it MUST set the "iss" to itself and
+    sign such Embedded Tokens with its own private key as described in the
+    [AAI Specification](https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md).
+
+8.  <a name="requirement-8"></a> Passport Visas are designed for machine
     interpretation only to make an access decision and is a non-goal to support
     rich user interface requirements nor do these claims fully encode the audit
     trail.
 
-9.  <a name="requirement-9"></a> An RI Claim Object MAY contain a
-    "[condition](#condition-optional-on-specific-ri-claims)" field that
-    restricts the RI Claim Object to only be valid when the condition is met.
+9.  <a name="requirement-9"></a> A Passport Visa Object MAY
+    contain a "[condition](#condition)" field that restricts the Passport Visa
+    to only be valid when the condition is met.
 
     -   For example, an identity can have several affiliations and a
-        ControlledAccessGrants RI Claim Object MAY be coupled to one of them
-        using the Condition field.
+        "ControlledAccessGrants" in a Passport Visa MAY be coupled to one
+        of them using the Condition field.
 
-10. <a name="requirement-10"></a> Processing a Passport within a Claims
+10. <a name="requirement-10"></a> Processing a Passport within a Passport
     Clearinghouse is to abide by the following:
 
-    1.  A Claims Clearinghouse MUST ignore all RI Claim Objects is does not need
-        to process a particular request and MUST ignore all RI Claim Objects
-        unless it explicitly has a sufficient trust relationship with the
-        "[source](#source-required)" of the RI Claim Object.
+    1.  A Passport Clearinghouse MUST ignore all Passport Visas is does not
+        need to process a particular request.
 
-    2.  Claims Clearinghouses SHOULD ignore claims that aren’t needed for their
-        purposes.
+    2.  Passport Clearinghouses MUST ignore Passport Visas unless:
+    
+        1.  The Passport Clearinghouse has a sufficient trust relationship
+            with the [Passport Assertion Source](#passport-assertion-source);
+            or
```

Comment:
The Clearing house should have a trust in both the Passport Assertion Source and the Passport Visa Signatory. Otherwise an evil Passport Visa Signatory may issue fake assertions pretending to be from a trusted Passport Assertion Source.

Reply:
Done